.github/stale.yml

```diff
@@ -1,31 +1,64 @@
-  it fixed in the near-future. Extending the lifetime of an issue
-  may cause the development team to prioritise it over other issues,
-  which may be closed as stale instead.
-
-# Comment to post when closing a stale issue. Set to `false` to disable
-closeComment: >
-  This issue has been closed automatically. If this still affects you please
-  re-open this issue with a comment or contact us so we can look into resolving it.
+
+# Options specific to issues
+issues:
+  # Number of days of inactivity before an issue becomes stale
+  daysUntilStale: 180
+
+  # Number of days of inactivity before a stale issue is closed
+  daysUntilClose: 7
+
+  # Issues with these labels will never be considered stale
+  exemptLabels:
+    - Induction
+    - Roadmap
+
+  # Comment to post when marking an issue as stale. Set to `false` to disable
+  markComment: >
+    This issue has been automatically marked as stale because it has not had
+    activity in 6 months. It will be closed in 7 days if no further activity occurs.
+
+    Allowing issues to close as stale helps us filter out issues which can wait
+    for future development time. All issues closed by stale bot act like normal issues; they can be searched for, commented on or reopened at any point.
+
+    If you'd like a closed stale issue to be considered,
+    feel free to either re-open the issue directly or contact a developer.
+
+    To extend the lifetime of an issue please comment below,
+    it helps us see that this is still affecting you and you want
+    it fixed in the near-future. Extending the lifetime of an issue
+    may cause the development team to prioritise it over other issues,
+    which may be closed as stale instead.
+
+  # Comment to post when closing a stale issue. Set to `false` to disable
+  closeComment: >
+    This issue has been closed automatically. If this still affects you please
+    re-open this issue with a comment or contact us so we can look into resolving it.
+
+pulls:
+  # Number of days of inactivity before a pull request becomes stale
+  daysUntilStale: 90
+
+  # Number of days of inactivity before a stale pull request is closed
+  daysUntilClose: 7
+
+  # Comment to post when marking an issue as stale. Set to `false` to disable
+  markComment: >
+    This pull request has been automatically marked as stale because it has not had
+    activity in 3 months. It will be closed in 7 days if no further activity occurs.
+
+    Allowing pull requests to close as stale helps us filter out old work that is no longer
+    relevant and helps developers focus on reviewing current work.
+
+    All pull requests closed by this bot act like normal pull requests; they can be searched for, commented on or reopened at any point.
+
+    This pull request has been closed automatically. If these changes are still relevant
+    then please re-open this pull request with a comment and tag \@mantidproject/gatekeepers
+    to highlight it needs to have a reviewer assigned.
```

Comment:
I think this should be in the next section or removed? At this point the PR is still in the "warning" phase

Reply:
Oh yes good spot. Copy-and-paste error! I think what I meant to keep was just
*"If these changes are still relevant
    then please comment and tag \@mantidproject/gatekeepers
    to highlight it needs to have a reviewer assigned."*

Is that worth keeping do you think?

Reply:
I think so, if people start abusing it for everything we can always remove that note too

Reply:
Thanks David. This should be corrected now.